iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -115,8 +115,8 @@ struct greater_or_equal
         Expects(t >= Minimum);
     }
```

Comment:
@saif-at-github Will you add the suppression or does something speak against this?

Reply:
@saif-at-github friendly ping :slightly_smiling_face:

Reply:
@saif-at-github This is still missing is, it not?

Reply:
Yes, it's still missing. I'm not insisting that the suppression is added now as there are more violations to be fixed in another PR anyway. But since you have to push another commit to fix clang-format you could also add it right now...

Reply:
@FerdinandSpitzschnueffler @MatthiasKillat  corrected.